_rules/html-page-lang-valid-bf051a.md

```diff
@@ -66,26 +66,18 @@ This `html` element has a `lang` attribute whose value is a [valid][valid langua
 This `html` element has a `lang` attribute whose value is not a [valid language subtag][].
 
 ```html
-<html lang="xyz"></html>
+<html lang="enn-US"></html>
 ```
 
 #### Failed Example 2
 
-This `html` element has a `lang` attribute whose value is not a [valid language subtag][]. Even though the [primary language subtag][] is valid, the [region subtag] is not.
-
-```html
-<html lang="en-US-GB"></html>
-```
-
-#### Failed Example 3
-
 This `html` element has a `lang` attribute whose value is not a [valid language subtag][].
 
 ```html
-<html lang="123"></html>
+<html lang="#1"></html>
```

Comment:
Why are you changing this?

Reply:
Changing according to https://github.com/act-rules/act-rules.github.io/issues/1097#issuecomment-573747130

Reply:
Example of an invalid value - numbers and non-alphanumeric characters are both invalid but don't necessarily need to be separate examples